docs/dev/searxng_extra/index.rst

```diff
@@ -1,14 +1,15 @@
 .. _searxng_extra:
 
-======================================================
-Tooling box ``searxng_extra`` for developers and users
-======================================================
+=============================
+Tooling box ``searxng_extra``
+=============================
 
-In the folder :origin:`searxng_extra/` we maintain some tools useful for
+In the folder :origin:`searxng_extra/` we maintain some tools useful for CI &
 developers.
```

Comment:
it's nicer if the line break is not in the middle of "CI & developers".

Reply:
> it's nicer if the line break is not in the middle of "CI & developers".

Yes, that true, but this is formatted by line size of 80 characters / I do not want to change the line size _here and there_, hope that is OK.

Reply:
It looks horrible ... I don't think we should blindly apply automatic line wrapping if this is the output.

Reply:
fixed